analytic_engine/src/sst/parquet/async_reader.rs

```diff
@@ -56,14 +56,20 @@ pub struct Reader<'a> {
     predicate: PredicateRef,
     batch_size: usize,
 
-    /// init those fields in `init_if_necessary`
+    /// Init those fields in `init_if_necessary`
     meta_data: Option<MetaData>,
     row_projector: Option<RowProjector>,
+
+    /// Options for `read_parallelly`
+    parallelism_options: ParallelismOptions,
```

Comment:
This fields is unnecessary  for `Reader`, it's only used in `read_parallel`, we can just a argument there.

Reply:
Maybe it is suitable to place it here?
+ `Reader` has the parallel read ability(by `read_parallelly`).
+ caller can pass `read_parallelism` as a suggest parallelism.
+ `Reader` check the suggest parallelism according to its `ParallelismOptions` generated in `new`.

But the name `ParallelismOptions` may be not a good name to express this?